src/Illuminate/Database/Migrations/MigrationCreator.php

```diff
@@ -209,6 +214,23 @@ protected function getDatePrefix()
         return date('Y_m_d_His');
     }
 
+    /**
+     * Get the equivalent primary key method.
+     *
+     * @return string
+     */
+    protected function getPrimaryKeyField($primary)
+    {
+        if ($primary === 'UUID') {
+            return 'uuid(\'id\')';
+        }
+        if ($primary === 'ULID') {
+            return 'ulid(\'id\')';
+        }
+
+        return 'id()';
```

Comment:
why not just utilise `Illuminate\Database\Schema\Blueprint::$defaultMorphKeyType`

Reply:
thanks @crynobone , I will look into utilising that instead

Reply:
@crynobone, how do you suggest the `Blueprint::$defaultMorphKeyType` be used in this section?